plugins/node/opentelemetry-instrumentation-undici/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './undici';
```

Comment:
It'd be good to export explicitly here instead of using `export *`, we're trying to move away from using `export *` as it sometimes ends up making internal stuff part of the public interface.

```suggestion
export * from './undici';
```

Reply:
I'll give it a try and you give me feedback about the result :)

Reply:
done!